### PR TITLE
CLOUD-45606 make cdb doctor docker-machine compatible

### PR DESCRIPTION
**Pull request by sodre90**
@lalyos @mhmxs review pls

### CODE REVIEW

**Comment by lalyos** [comment]
please add a docker-machine section into dev-docs/cloudbreak-development.md

File: include/docker.bash [hunk 1]
```diff
@@ -59,6 +59,30 @@ UNTIL-BOOT2DOCKER-CLI-366-GET-MERGED
     info "boot2docker: OK" | green
 }
 
+docker-check-docker-machine() {
+    docker-machine version &> /dev/null || local missing=1
+    if [[ "$missing" ]]; then
```
Comment (lalyos) [inline include/docker.bash]: these 2 lines should be just 1:
```
if docker-machine version &> /dev/null; then
```

File: include/docker.bash [hunk 2]
```diff
@@ -59,6 +59,30 @@ UNTIL-BOOT2DOCKER-CLI-366-GET-MERGED
...
+        error "docker-machine command not found, please install by:"
+        echo "  brew install docker-machine" | blue
+        _exit 127
+    fi
+    if [[ "$(docker-machine status)" == "Running" ]]; then
+      if [[ "$(docker-machine active)" == "$DOCKER_MACHINE" ]]; then
+          info "docker-machine env init: OK"
+      else
+          error "docker-machine env is not set correctly, please run:"
+          echo " eval \$(docker-machine env $DOCKER_MACHINE)" | blue
```
Comment (lalyos) [inline include/docker.bash]: if you dont want bash to evaluat $VARS, just use '' instead of ""

**Comment by sodre90** [comment]
docker-machine section added into dev-docs/cloudbreak-development.md

File: include/docker.bash (hunk 1 above)
Comment (sodre90) [inline include/docker.bash]: fixed

**Pull request merged**